sdk/keyvault/Azure.Security.KeyVault.Keys/tests/ManagedHsmCryptographyClientLiveTests.cs

```diff
@@ -46,9 +46,12 @@ public async Task EncryptLocalDecryptOnManagedHsm([EnumValues(
             nameof(EncryptionAlgorithm.A128Cbc),
             nameof(EncryptionAlgorithm.A192Cbc),
             nameof(EncryptionAlgorithm.A256Cbc),
+            nameof(EncryptionAlgorithm.CkmAesKeyWrap),
```

Comment:
This is an awkward placement.  If ordering is not important, can we move this to the end to preserve alphabetical/numerical ordering?